### ISSUE TITLE
Split data and apply resampling to balance data

### ISSUE THREAD

**Issue by adellegia** [adellegia/predicting-merger-decision-outcomes#4]
- [x] option1: resample training data to oversample minority class
- [x] option2: balance data by undersampling majority cases
- [x] if option2: put excess to test set
- [x] put simplified cases to test set
